app/src/shared/components/Checkbox/index.jsx

```diff
@@ -1,28 +1,61 @@
-// @flow
-
 import React from 'react'
-import classNames from 'classnames'
+import styled, { css } from 'styled-components'
+import ImageChecked from './checkbox-checked.svg'
+import ImageUnchecked from './checkbox.svg'
+
+const Dummy = styled.div`
```

Comment:
`Dummy` is a bit weird name, how about `Tick`?

Reply:
Maybe a bit weird, maybe. `Tick` is fine. I'l' change.